### PR TITLE
To Do list: interactive list

### PR DESCRIPTION
**Pull request by ramiaquista**
## What I have done: 🔨
- Added task-status.js file with method boxNotMarked() to check if the box is marked.
- Added event listener to the checkbox('change') to make the task mark or unmarked.
- Added styling to each task that it's marked as completed.
- Add dynamically update of the completed key of each task object.
- Add localStorage functionality to handle previous data when the page is reloaded.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -1,35 +1,50 @@
...
+const dataStorage = JSON.parse(localStorage.getItem('todoTasks'));
+
+let task1 = {
   description: 'Go to the supermarket',
   completed: false,
   index: 0,
 };
 
-const task2 = {
+task1 = dataStorage[0];
```
Comment (rukundoeric) [inline src/index.js:14]: - [ ] <b>REQUIRED:</b> I tried to run the app, but it render an empty page with the following error in the console. kindly fix it.
Screen Shot 2021-09-22 at 21 12 00

**Review by rukundoeric — CHANGES_REQUESTED** [review]
Hi @ramiaquista, 

_Great job so far 👏 💯 👍. However, there are few issues we need to address before we move on. Please, check the comments below._
## Highlights
  - Follows git workflow.  💯
  - Codebase looks clean and professional. 💯 

## Required Changes ♻️
  - [ ] Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
  - [ ] In the package.json in dependencies, you have a lot of packages that are not used anywhere in your project, which makes them useless, kindly remove all useless packages from your package.json.
  - [ ] Check the comments under the review.

Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
> As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/index.js (hunk 1 above)
Comment (ramiaquista) [reply src/index.js]: Hi @rukundoeric , thanks! I didn't notice that before, I have solved it. Could you help me and tell me if know it's running correctly and properly before submitting another pull request review! Only if you have time, don't worry!

Thank's for your time! 😄

Comment (rukundoeric) [reply src/index.js]: Sorry, I didn't get the message on time. but no worries I'm reviewing it.

**Review by rukundoeric — APPROVED** [review]
Hi @ramiaquista,

_Excellent job on this project. Keep it up 🎆_

_Congratulations! 🎉  on the splendid job 👍
You can now merge this PR and submit the project completion form 🚀 . I wish you all the very best in your next tasks 🥇_1


Cheers and Happy coding!👏👏👏

__Feel free to leave any questions or comments in the PR thread if something is not 100% clear.__

------
>As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**